Sources/AWSLambdaRuntimeCore/LambdaConfiguration.swift

```diff
@@ -64,7 +64,6 @@ extension Lambda {
         struct RuntimeEngine: CustomStringConvertible {
             let ip: String
             let port: Int
-            let keepAlive: Bool
```

Comment:
I have removed the `keepAlive` option here, since neither tested nor used anywhere in the project. The HTTPClient should always prefer `keep-alive` but should be able to close a connection after a `connection: close` header. There is a test that ensures that the HTTPClient works correctly when a `connection: close` header is received.

Reply:
I think the intent was to be able and force closed connection by sending that header from the client, but this is fine.